charts/openyurt/templates/pool-coordinator.yaml

```diff
@@ -0,0 +1,205 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: pool-coordinator-apiserver
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
```

Comment:
we need to add service topology annotation for pool-coordinator service.

Reply:
done